.circleci/unittest/linux/scripts/setup_env.sh

```diff
@@ -38,10 +38,7 @@ conda activate "${env_dir}"
 
 # 3. Install Conda dependencies
 printf "* Installing dependencies (except PyTorch)\n"
-FFMPEG_PIN="=4.2"
-if [[ "${PYTHON_VERSION}" = "3.9" ]]; then
-    FFMPEG_PIN=">=4.2"
-fi
+FFMPEG_PIN=">=4.2"
```

Comment:
Linux seems to throw [SegFault](https://app.circleci.com/pipelines/github/pytorch/vision/20813/workflows/861b0223-f09c-4b57-ab30-6fde18bedbfd/jobs/1692903) if we install 4.3

FYI @jdsgomes